Sources/SwiftDocC/Model/Rendering/Symbol/AvailabilityRenderMetadataItem.swift

```diff
@@ -85,7 +82,7 @@ public struct AvailabilityRenderItem: Codable, Hashable, Equatable {
     public var unconditionallyUnavailable: Bool?
     
     /// If `true`, the symbol is introduced in a beta version of this platform.
-    public var isBeta: Bool?
+    public let isBeta: Bool?
```

Comment:
The reason I made this change was so we wouldn't miss the need to initialize `isBeta` if we added a new initializer to this struct, to avoid this issue repeating in the future.

However happy to revert it to avoid a source breaking change.

Reverted in 5858ded23b74d21d4ac1ed69b48df92a12747542